masterlist.yaml

```diff
@@ -17344,16 +17344,26 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Arena - An Encounter Zone Overhaul' ]
         condition: 'active("Arena.esp") and not active("DeadMansRefugeArenaPatch.esp")'
+    tag: [ C.Location ]
```

Comment:
The tag is already in the plugin's description, so it doesn't need to be added by LOOT:

```suggestion
```